react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util';
+
+import { reconnectNow } from '../functions';
+import ReloadButton from './ReloadButton';
+
+declare var AJS: Object;
+declare var APP: Object;
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
```

Comment:
I voted in @hristoterezov's poll on HipChat and I'm copying here in the context for the sake of completeness: I prefer `const logger = require(`; otherwise, we end up with 2 disjoined lines (because the import has to be separated from the `const logger` with at least 1 line and 2 other import groups will likely be in between most of the time).